deployments/structs.go

```diff
@@ -14,29 +14,16 @@
 
 package deployments
 
-//go:generate stringer -type=DeploymentStatus structs.go
+//go:generate go-enum -f=structs.go
 
-// DeploymentStatus represent the current status of a deployment
+// DeploymentStatus x ENUM(
+// INITIAL,
+// DEPLOYMENT_IN_PROGRESS,
+// DEPLOYED,
+// UNDEPLOYMENT_IN_PROGRESS,
+// UNDEPLOYED,
+// DEPLOYMENT_FAILED,
+// UNDEPLOYMENT_FAILED,
+// SCALING_IN_PROGRESS
+// )
```

Comment:
Minor issue but go convention is to use MixedCaps instead of underscores, and code climate is raising issues on this.
As these enums will be prefixed by DeploymentStatus, could we have somzthing like :

Initial,
InProgress,
Deployed,
Undeploying,
Undeployed,
Failed,
UndeploymentFailed,
ScalingInProgress

Reply:
Yes I agree

Reply:
But... it impacts the generated string method. I see what is possible.

Reply:
I just can remove the DeploymentStatus prefix as before. WYT ?

Reply:
ok